### DISCUSSION TITLE
Vaccine Tracking

### DISCUSSION THREAD

**Discussion by five2seven** [babybuddy/babybuddy#486] [Ideas]
Any thoughts about adding a way to track vaccine schedules?

**Comment by cdubz** [comment]
Somewhat related issues are #217, #408, #153, and #66 -- none of which have got enough traction for feature work.

One thing you can do with the introduction of tags is use notes and tags to keep track on this. Here's an example of how that might look (I just mocked this up in the demo instance):

![image](https://user-images.githubusercontent.com/10456740/173844180-38dd3801-02f1-4292-867b-b23ff17e2012.png)

**Reply by jwildman16** [reply]
> Somewhat related issues are #217, #408, #153, and #66 -- none of which have got enough traction for feature work.

How would I increase the traction on these? Just comment agreeing that it would be useful? I was also looking for things like vaccination tracking. Thanks!

As an aside, have you seen the Baby Connect app on Android? It has categories for vaccinations, doctor's visits, etc. We used it previously and liked most of its core features. Worth checking out if you haven't seen it.

**Reply by cdubz** [reply]
Well #217 is likely the most valuable feature of that group for its flexibility. The functionality is pretty well laid out there so “traction” in that case would essentially be a PR. I’m willing to review and help someone along with this but I don’t have the time to devote to building it out myself currently. I’m also frankly still somewhat skeptical of the value over notes + tags.

And if #217 doesn't sound like what you’re interested in then yea please do feel free to add your comments to one of the other relevant issues. None of the other issues mentioned are as far along w.r.t. details of the requested feature.

**Reply by jwildman16** [reply]
Thank you for the info and for your time on this!
